test/get_previous_releases.py

```diff
@@ -136,6 +142,9 @@ def download_binary(tag, args) -> int:
         platform = "osx64"
     tarball = 'bitcoin-{tag}-{platform}.tar.gz'.format(
         tag=tag[1:], platform=platform)
+    if platform == "win64" and tag >= "v22.0":
```

Comment:
Why is this restricted to 22.0 and above? The naming scheme for the Windows zips hasn't changed.